sdks/java/io/amazon-web-services/src/test/java/org/apache/beam/sdk/io/aws/s3/S3FileSystemTest.java

```diff
@@ -64,6 +77,28 @@
 /** Test case for {@link S3FileSystem}. */
 @RunWith(JUnit4.class)
 public class S3FileSystemTest {
+  private S3Mock api;
+  private AmazonS3 client;
+
+  @Before
+  public void before() {
+    api = new S3Mock.Builder().withPort(8001).withInMemoryBackend().build();
+    api.start();
+
+    EndpointConfiguration endpoint =
+        new EndpointConfiguration("http://localhost:8001", "us-west-2");
+    client =
+        AmazonS3ClientBuilder.standard()
+            .withPathStyleAccessEnabled(true)
```

Comment:
Is this pathStyleAccess configuration mandatory if not please remove it (I ask this because there is another ongoing PR to validate this).